### DISCUSSION TITLE
Trouble building on Termux

### DISCUSSION THREAD

**Discussion by nerdhardest** [uutils/coreutils#4715] [General]
FYI, 'Termux' is a command-line application for Android devices
(phones, etc.) that runs on both 32-bit arm and 64-bit aarch64
Android devices.

I've done a lot of Rust compilations under Termux, so it's reasonably
well tested.

However, I wasn't able to get the Rust coreutils to build on either
a 32-bit or 64-bit Termux device.

E.g., I couldn't build 'onig_sys' on either a 32-bit or a 64-bit Termux
device; I could build 'blake3' on a 32-bit, but not a 64-bit Termux
device.

I was, however, able to build 'uu_factor' on a 32-bit Termux
device, so it appears that the problem should be relatively
easy to fix.

(These Termux devices have plenty of RAM, so this Termux
problem is different from the RAM problem I described in
a different topic.)

**Comment by tertsdiepraam** [comment]
The error messages would be helpful here. `onig_sys` might be due to an incompatibility of `onig` with clang 16. There's already an issue for this. Feel free to open individual issues for the rest!

Edit: There's only an issue on `findutils` for `onig`. I'd be nice to have one here too.
